src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.DataConverter;
+import service.FileReader;
+import service.FileWriter;
+import service.ReportGenerator;
+import service.ShopService;
+import service.impl.DataConverterImpl;
+import service.impl.FileReaderImpl;
+import service.impl.FileWriterImpl;
+import service.impl.ReportGeneratorImpl;
+import service.impl.ShopServiceImpl;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+import strategy.impl.BalanceOperation;
+import strategy.impl.OperationStrategyImpl;
+import strategy.impl.PurchaseOperation;
+import strategy.impl.ReturnOperation;
+import strategy.impl.SupplyOperation;
+
+public class Main {
+    private static final String INPUT_FILE = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/finalReport.csv";
+
+    public static void main(String[] args) {
```

Comment:
The `main` method seems to be doing too much. Consider refactoring the code to separate the initialization of services and handlers from the main business logic.